### ISSUE TITLE
Get-HPOVServer fails with DuplicateKeysInJsonString

### ISSUE THREAD

**Issue by andrevdn** [HewlettPackard/POSH-HPOneView#407]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
Normal output of Get-HPOVServer of all SY480 nodes including SY 480 Gen10 w/ PCIe Exp nodes with NVIDIA Tesla M10 cards.

### Actual Behavior
After running Get-HPOVServer 
```
Get-HPOVServer : Cannot convert the JSON string because a dictionary that was converted from the string contains the duplicated keys 'NVIDIA TESLA M10' and 'NVIDIA Tesla M10'.
At line:1 char:1
+ Get-HPOVServer -verbose
+ ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Get-HPOVServer], InvalidOperationException
    + FullyQualifiedErrorId : DuplicateKeysInJsonString,Get-HPOVServer
```

### Steps to reproduce
When a SY480 node has NVIDIA Tesla M10 cards with different names. (Tesla and TESLA). Run Get-HPOVServer

### Version Information
```
hpov                                LibraryVersion                                           
------------------                  -------------- ----                                           
ApplianceVersion: 4.20.01.385523.01 4.20.1971.1960 
```

# Output from $PSVersionTable
```
Name                           Value                                                                                                                                                         
----                           -----                                                                                                                                                         
PSVersion                      5.0.10105.0                                                                                                                                                   
WSManStackVersion              3.0                                                                                                                                                           
SerializationVersion           1.1.0.1                                                                                                                                                       
CLRVersion                     4.0.30319.42000                                                                                                                                               
BuildVersion                   10.0.10105.0                                                                                                                                                  
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                       
PSRemotingProtocolVersion      2.3     
```

**Comment by ChrisLynchHPE** [comment]
Can you supply the captured output from Get-HPOVCommandTrace?  I'm interested in the JSON reply from the API that will be captured.

**Comment by andrevdn** [comment]
Attached the requested info.

**Comment by andrevdn** [comment]
[1907021538_HPOV_transcript.log](https://github.com/HewlettPackard/POSH-HPOneView/files/3356352/1907021538_HPOV_transcript.log)

**Comment by ChrisLynchHPE** [comment]
This has been addressed in the newer 4.20 (fixed in the [4.20.2133.3186 release](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v4.20.2133.3186)) and 5.00 PowerShell libraries.  Please upgrade you library.  Apologies for not tagging this issue to the release, or notifying you.
